pulser-pasqal/pulser_pasqal/pasqal_cloud.py

```diff
@@ -104,7 +107,7 @@ def _serialize_seq(
             return seq.serialize()
```

Comment:
Do you happen to know if the QPU has moved to the abstract representation? If so, this could be a good opportunity to change this part

Reply:
I believe not, but will check in the morning.

Reply:
I don't see any blockers for not using the abstract representation, all our stack up to the QPU execution is compatible with it

Reply:
made into always abstract